linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Process info for topline summary
+ */
+var ProcInfo = (function() {
+
+  var msToStr = new MsToStringConverter();
+  var bytesToStr = new BytesToStringConverter();
+  var refreshUri = "/admin/metrics";
+
+  function pretty(name, value) {
+    switch (name) {
+      case "jvm/uptime": return msToStr.convert(value);
+      case "jvm/mem/current/used": return bytesToStr.convert(value);
+      case "jvm/gc/msec": return msToStr.convert(value);
+      default: return value;
+    }
+  }
+
+  function render(data) {
+    var json = $.parseJSON(data);
+    _(json).each(function(obj) {
+      var id = obj.name.replace(/\//g, "-");
+      var value = pretty(obj.name, obj.value);
+      $("#"+id).text(value);
+    });
+  }
```

Comment:
I don't think this needs to be done now, but it'd be nice if in the future this used process_info.template